### PR TITLE
Show last revision name as current review name

### PR DESCRIPTION
**Pull request by praveenkuttappan**

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Client/css/site.scss [hunk 1]
```diff
@@ -26,6 +26,10 @@ a.navbar-brand
     word-break: break-all;
 }
 
+tr:nth-child(even) {
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Client/css/site.scss:29]: unrelated?

**Review by pakrym — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Client/css/site.scss (hunk 1 above)
Comment (pakrym) [reply src/dotnet/APIView/APIViewWeb/Client/css/site.scss:29]: https://getbootstrap.com/docs/4.0/content/tables/#striped-rows

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Client/css/site.scss:29]: nope. Included one  of the suggestion from Jonathan to show striped table.

Comment (JonathanGiles) [reply src/dotnet/APIView/APIViewWeb/Client/css/site.scss:29]: Bootstrap can do this in a simpler way. I'd suggest reverting this and I'll create a PR that demonstrates a few style changes I would propose.

Comment (JonathanGiles) [reply src/dotnet/APIView/APIViewWeb/Client/css/site.scss:29]: I've just quickly done a few of my suggested 'easy wins' here: https://github.com/Azure/azure-sdk-tools/pull/1751
Feel free to take the changes and apply them into a proper PR if you want.

Comment (JoshLove-msft) [reply src/dotnet/APIView/APIViewWeb/Client/css/site.scss:29]: The striped tables get messed up when using the search filter:
image
